docker/Dockerfile

```diff
@@ -25,13 +25,23 @@ RUN \
 	&& apt-get update \
 	&& apt-get install -y --no-install-recommends \
 		libpango1.0-dev \
-		libcairo2-dev \
 		libmagickwand-dev \
 		ghostscript \
 		imagemagick \
 		gsfonts
 
 RUN \
+	echo "Install newer version of Cairo" \
+	&& curl -O https://www.cairographics.org/releases/cairo-1.14.8.tar.xz \
+	&& tar xf cairo-1.14.8.tar.xz \
+	&& cd cairo-1.14.8 \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& cd .. \
+	&& rm -r cairo-1.14.8 cairo-1.14.8.tar.xz
```

Comment:
tthis could do with a bit of cleanup - suggestions

* can curl without `-O`, then pipe the result straight into `curl {url} | tar x`
* best practice is not to use `cd` in RUN commands, use `WORKDIR` instead - https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#workdir 
  - then `WORKDIR /` after to go back
* i'd possibly `WORKDIR /tmp/` before this, that way we don't need the `rm` step as we remove `/tmp/*` in the next step